### DISCUSSION TITLE
Middleware error StartSession::addCookieToResponse()

### DISCUSSION THREAD

**Discussion by make-skibidi-brainrot** [filamentphp/filament#11787] [Help]
### Package

Panel builder

### Package Version

v3.2.46

### How can we help you?

An error encountered in the Filament admin panel installed on http://localhost. The error occurs when a I was attempts to access the root URL (http://localhost) while not logged in.

ERRPR MESSAGE:
```
Illuminate\Session\Middleware\StartSession::addCookieToResponse(): Argument #1 ($response) must be of type Symfony\Component\HttpFoundation\Response, null given, called in D:\Projects\Web\teamloser\vendor\laravel\framework\src\Illuminate\Session\Middleware\StartSession.php on line 1
```
If I manually visit http://localhost/login and logged in then if I go to my root route there is no error

**Comment by ArtMin96** [comment]
I think it has nothing to do with Filament because you need to go to the following URL `/admin/login` to log in.
